nsv13/code/modules/overmap/types/solgov.dm

```diff
@@ -56,6 +56,14 @@
 	starting_system = "Staging" //Required for all player ships
 	armor = list("overmap_light" = 99, "overmap_medium" = 50, "overmap_heavy" = 25)
 
+/obj/structure/overmap/nanotrasen/solgov/vnc
```

Comment:
You have not added the mas of the ship or the bound height/width. This is important to determining the hitbox and based on your description, this ship is supposed to be smaller and faster.